### PR TITLE
PP 73 Readme file

### PR DESCRIPTION
**Pull request by vrudnitskaya**
### Description
The readme file for the frontend repository has been added. It contains basic information about our project.

### CODE REVIEW

**Review by ofeterovskaya — APPROVED** [review]
Thank you Valya.
What do you think. Do we need to add ?
.env
VITE_API_BASE_URL=http://localhost:8000/api/v1
VITE_FRONTEND_URL=http://localhost:5173
VITE_CLIENT_ID=your_ClienT_ID

**Comment by vrudnitskaya** [comment]
> Thank you Valya. What do you think. Do we need to add ? .env VITE_API_BASE_URL=http://localhost:8000/api/v1 VITE_FRONTEND_URL=http://localhost:5173 VITE_CLIENT_ID=your_ClienT_ID

I'll add this.

File: README.md [hunk 1]
```diff
@@ -1,30 +1,150 @@
...
+  npm run preview
+
+This configuration utilizes modern technologies and tools to optimize the development process and boost the application's performance.
+
+## Key Features
+
+### Effortless Registration and Profile Creation
+
+- **Sign Up**: Users can sign up as either a student or a teacher. After completing the registration, they will be redirected to the login page. Upon successful login, users gain full access to the dashboard. In case of a forgotten password, they can request a reset link via email and log in with a newly created password.
+- **Edit Profile**: Users can update their personal information, upload the their photo.
```
Comment (JoHeyJo) [inline README.md:87]: "upload their photo"

**Review by JoHeyJo — APPROVED** [review]
Looks good!

**Pull request merged**